ssl/tls13_enc.c

```diff
@@ -257,13 +257,17 @@ int tls13_generate_master_secret(SSL *s, unsigned char *out,
 size_t tls13_final_finish_mac(SSL *s, const char *str, size_t slen,
                              unsigned char *out)
 {
-    const char *mdname = EVP_MD_get0_name(ssl_handshake_md(s));
+    const EVP_MD *md = ssl_handshake_md(s);
+    const char *mdname = EVP_MD_get0_name(md);
```

Comment:
Don't we need a NULL check here?

Reply:
It looks like EVP_MD_get0_name returns NULL if its argument is NULL, so this should be OK.

Reply:
Safer to check it nonetheless.  There is no telling what changes will be made in the future that break this assumption.

For example, `EVP_MD_CTX_get_size()` never used to return a failure -- it could as of 3.0.